### PR TITLE
Add action to set mealplan

### PR DESCRIPTION
**Pull request by joostlek**
## Proposed change

Add service to set mealplan

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/122317
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Introduced two new actions for the Mealie integration: 
    - `mealie.set_mealplan` for setting a customizable meal plan for specific dates.
    - `mealie.set_random_mealplan` for generating a random meal plan for selected dates, enhancing meal planning flexibility.

### CODE REVIEW

File: source/_integrations/mealie.markdown [hunk 1]
```diff
@@ -83,6 +85,19 @@ Import the recipe into Mealie from a URL.
...
 
+### Action `mealie.set_mealplan`
+
+Set a mealplan on a specific date.
+
+| Data attribute    | Optional | Description                                         |
+|-------------------|----------|-----------------------------------------------------|
+| `config_entry_id` | No       | The ID of the Mealie config entry to get data from. |
+| `date`            | No       | The date that should be filled.                     |
+| `entry_type`      | No       | One of "breakfast", "lunch", "dinner" or "side".    |
```
Comment (c0ffeeca7) [inline source/_integrations/mealie.markdown]:
```suggestion
| `entry_type`      | No       | One of "breakfast", "lunch", "dinner", or "side".    |
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @joostlek :+1:

**Comment by joostlek** [comment]
Oh there is also one at 109

**Comment by c0ffeeca7** [comment]
> Oh there is also one at 109

yes, but that one is too far outside for me to comment on.

**Comment by joostlek** [comment]
If I have the time and room I'll update that. I want to do that before merging to avoid a second PR for that tiny change

**Comment by frenck** [comment]
Shit missed the last one... so I guess another PR is needed later on 🙏

**Pull request merged**